### ISSUE TITLE
Gen 1 - 3 Support

### ISSUE THREAD

**Issue by DocLevCrav** [hsahovic/poke-env#265]
I am currently using poke-env to create a bot to fight in gen 1 battles, because they have the least amount of mechanics. This is just an enhancement, but I hope this can be fixed so that I can improve my bot.

**Comment by hsahovic** [comment]
Hey @DocLevCrav ! Have you tried using poke-env with gen 1?

**Comment by DocLevCrav** [comment]
Yes I have, and when I do so it says that it uses Gen 4 data instead of Gen 1, because it is not available. It works, I just hope there will eventually be complete Gen 1 - 3 support

**Comment by cameronangliss** [comment]
Aside from PP tracking, which is very difficult to get right in gens 1-3 due to various in-game bugs that are replicated in showdown and unannounced Pressure activation in gen 3, the battles are well-tracked in gens 1-3 now, as is evidenced in the strict battle tests we have in the nightlies. Additionally, we now have gen 1-3 data in poke_env/src/data/. Closing now.
